client/src/utils/countdown.ts

```diff
@@ -0,0 +1,1434 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
Is there a way we can import this using yarn instead of adding the full file?

Reply:
I tried using the proper NPM repo, found here https://github.com/mckamey/countdownjs, but it hasn't been updated in a little while. (That's fine for the Date computation part, but not fine for the module loading part). I spent a couple of hours trying to get Webpack to play nice with it, and finally gave up, using this solution ... but I'm not the savviest when it comes to Webpack, so if you have any alternatives, I'm all ears!

Reply:
Probably not worth it to try to get it to work nicely. I think it works for now, but I think it's good to document that this is the case for maintenance.